integration/rocket-chat/main.js

```diff
@@ -525,8 +525,17 @@ function SmartiWidget(element,_options) {
             resultCount.empty();
             resultPaging.empty();
             loader.show();
+
+            let customSuffix = RocketChat.settings.get('Assistify_AI_DBSearch_Suffix') || '';
```

Comment:
OK, you're totally right! But then we need some configuration options that will enable us to set different params for different clients/customers ... So this issue is dependent on a multi-client configuration concept for smarti.

For now I would suggest to reject that change and reopen it, when we know how to handle client configurations in general.